examples/api/src/App.svelte

```diff
@@ -1,6 +1,6 @@
 <script>
   import { onMount } from "svelte";
-  import { open } from "@tauri-apps/api/shell";
+  import { open } from "@tauri-apps/open";
```

Comment:
how did this work? that's not a package name

Reply:
Apologies, any kind of instructions to run the /examples would be very advantageous. So far I've figured out you need to use yarn over npm because of the "link" property. It seems you are saying build should be run before dev, that isnt typical in javascript projects would be great to add that also. Thanks for your valuable time sir.

Reply:
Build should be run on the `api` npm package, not the api example. but yeah i agree it needs some instructions.